authors/apps/authentication/tests/tests.py

```diff
@@ -0,0 +1,138 @@
+import json
```

Comment:
@mnswaleh We can remove this file, since we've moved its contents to `tests_api.py`